.github/workflows/production.yml

```diff
@@ -83,3 +83,17 @@ jobs:
           K8S_SECRET_DATABASE_PASSWORD: ${{ secrets.K8S_SECRET_DATABASE_PASSWORD_STABLE }}
           VAULT_JWT_AUTH_PATH: ${{ github.event.repository.name }}-stable
           VAULT_KV_SECRET_MOUNT_POINT: stable
+
+      - name: Deploy Cronjob to send removal notifications and clean old data
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: send-reminders-and-clean-old-data-cronjob
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+          image_tag: ${{ github.sha }}
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW }}
+          target_namespace: ${{ secrets.K8S_NAMESPACE_STAGING }}
```

Comment:
STABLE

Reply:
🤦 Copypasted the configuration from staging, forgot to change that.

Reply:
Fixed it now.